src/Discord/functions.php

```diff
@@ -38,3 +38,15 @@ function mentioned($part, Message $message)
         return strpos($message->content, $part) !== false;
     }
 }
+
+/**
+ * Calculates UNIX timestamp from snowflake.
+ *
+ * @param STRING $snowflake The snowflake
+ *
+ * @return UNIX timestamp
+ */
+function timestampFromSnowFlake(string $snowflake)
+{
+    return (($snowflake >> 22) + 1420070400000);
```

Comment:
This constant will also need to be a string as it is more than the PHP max integer size on 32-bit PHP so we need to find a way to calculate timestamps without using it.